README.md

```diff
@@ -1,62 +1,76 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Awesome Books ES6
 
+> Awesome books is an app where you can add and remof ove list books you are currently reading. This app is built with the latest ES6 technology.
```

Comment:
- [ ] Kindly fix the typographical error here, you have `remof ove` instead of `remove`.